<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tdc</class>
 <widget class="QWidget" name="tdc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_tdc_lifetime_statistics" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QLabel" name="lifetime_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lifetime</string>
    </property>
   </widget>
   <widget class="QLabel" name="mean_lifetime_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean (µs)</string>
    </property>
   </widget>
   <widget class="QLabel" name="rms_lifetime_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RMS (µs)</string>
    </property>
   </widget>
   <widget class="QLabel" name="last_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Last (µs)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_mean_lifetime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>25</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_rms_lifetime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>55</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_last_lifetime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>85</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tdc_file_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>650</y>
     <width>521</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tdc_file_name">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>680</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_tdc_event_statistics" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QLabel" name="events_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Events</string>
    </property>
   </widget>
   <widget class="QLabel" name="rate_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate (Hz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mean_arrival_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean Arrival (s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="rms_arrival_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RMS Arrival (s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_rate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>25</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_mean_arrival">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>55</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_rms_arrival">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>85</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_tdc_total">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>115</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget_tdc_channel" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_tdc_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Ch.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tdc_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Ch.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tdc_start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tdc_stop">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>75</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="tdc_channel_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TDC Channel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>655</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="qcp_tdc" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>571</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_tdc_trigger" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QLabel" name="tdc_trigger_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TDC Trigger</string>
    </property>
   </widget>
   <widget class="QLabel" name="window_width_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Width (ns)</string>
    </property>
   </widget>
   <widget class="QLabel" name="window_offset_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Offset (ns)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tdc_window_width">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>35</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>51200</number>
    </property>
    <property name="singleStep">
     <number>25</number>
    </property>
    <property name="value">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tdc_window_offset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>65</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-51200</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>25</number>
    </property>
    <property name="value">
     <number>-10025</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_tdc_histogram_config" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_tdc_tmin">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>35</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_tmax_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>t max (ns)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nbins_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>n bins</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tdc_nbins">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>95</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_tdc_tmax">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>65</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>150000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
    <property name="value">
     <double>10000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_tmin_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>t min (ns)</string>
    </property>
   </widget>
   <widget class="QLabel" name="histo_param_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>20</width>
     <height>661</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_width">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_config">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>read conf.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_trigger">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Trigger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_cont">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Cont.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_offset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tdc_start_run">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tdc_stop_run">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tdc_clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_tdc_rotary_switches">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>45</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>60928</number>
   </property>
   <property name="displayIntegerBase">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tdc_firwmare_revision">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>75</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tdc_clear_tdcs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear TDCs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Firmware Rev.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tdc_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotary Switches</string>
   </property>
  </widget>
  <widget class="QLabel" name="board_tdc_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Board Panel</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tdc_log">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>285</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Log Scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_plot">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>285</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Plot</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
